app/src/main/java/org/mozilla/focus/exceptions/ExceptionsListFragment.kt

```diff
@@ -105,29 +107,35 @@ open class ExceptionsListFragment : BaseSettingsLikeFragment(), CoroutineScope {
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View = inflater.inflate(R.layout.fragment_exceptions_domains, container, false)
+    ): View {
+        FragmentExceptionsDomainsBinding.inflate(inflater, container, false)
+        return binding.root
```

Comment:
Seems like the binding property is not initialized?
It will also need to be set to null in `onDestroyView`.